src/Swashbuckle.Swagger/project.json

```diff
@@ -1,27 +1,28 @@
 ﻿{
```

Comment:
I kinda liked the existing layout for the project.json files :( I'd prefer to have properties grouped by meaning as opposed to alphabetically. Any chance we could change these back?